libmscore/excerpt.cpp

```diff
@@ -1034,6 +1079,9 @@ QList<Excerpt*> Excerpt::createAllExcerpt(MasterScore *score)
             if (part->show()) {
                   Excerpt* e   = new Excerpt(score);
                   e->parts().append(part);
+                  e->tracks().insert(part->startTrack(), 0);
+                  e->tracks().insert(part->startTrack() + 1, 1);
+                  e->tracks().insert(part->startTrack() + 3, 2);
```

Comment:
If I get it right, this is about extracting the 4 voices of a staff, if so, voice 2 seems to be missing.
And we'd better use a for loop over VOICES.